guia/closures.md

```diff
@@ -1 +1,131 @@
-# Closures
+## Closures
+
+*Closures* são blocos de funcionalidade independentes que podem ser passados e usados em seu código. *Closures*, em Swift, são similares aos blocos em C e Objective-C e a *lambdas* em outras linguagens de programação.
+
+*Closures* podem capturar e armazenar referências para quaisquer constantes e variáveis do contexto no qual ele foi definido. Isso é conhecido como *closure* sobre constantes e variáveis. O Swift se encarrega de realizar o gerenciamento de memória de captura por você.
+
+>NOTA
+
+>Não se preocupe se você não estiver familiarizado com o conceito de captura. Isso é explicado em detalhes adiante em [Capturando Valores](./closures.md#capturing_values).
+
+Funções globais e aninhadas, como apresentado em [Funções](./functions.md#functions), são na verdade casos especiais de *closures*. *Closures* possuem uma, de três formas:
+
+* Funções globais são *closures* que possuem um nome e não capturam nenhum valor
+* Funções aninhadas são *closures* que possuem um nome e podem capturar valores das funções as quais ela está inserida
+* Expressões de *closure* são *closures* sem nome, escritos com uma sintaxe leve, que podem capturar valores dos contextos ao seu redor
+
+As expressões de *closure* do Swift possuem uma sintaxe clara com otimizações que encorajam uma sintaxe curta e que evita mal entendidos em cenários comums. Essas otimizações incluem:
+
+* Dedução de parâmetros e tipos de valor de retorno a partir do contexto
+* Retorno implícito a partir de expressões de *closure* únicas
+* Argumentos com nomes simples
+* Sintaxe do *closure* de fuga - do inglês *Trailing closure syntax*
+
+###Expressões de ***closure***
```

Comment:
No original não _closure_ tem ênfase. Acho que pode remover o negrito e deixar só o itálico por ser uma expressão em outro idioma.